core/metadata/metadata.go

```diff
@@ -0,0 +1,92 @@
+//
```

Comment:
IMHO all core patches should be sent to core repos first - metadata is still not integrated :(. I am not sure that Dell team is reviewing this repo yet.